api/src/identity-access-management/domain/usecases/authenticate-user.js

```diff
@@ -45,6 +47,17 @@ const authenticateUser = async function ({
       await userRepository.update({ id: foundUser.id, locale: foundUser.locale });
     }
 
+    const userLogin = await userLoginRepository.findByUserId(foundUser.id);
+
+    if (userLogin?.shouldSendConnectionWarning()) {
```

Comment:
Pourquoi l'utilisation de l'opérateur `?.` sachant que `shouldSendConnectionWarning` est une méthode de la classe `UserLogin` ?

```suggestion
    if (userLogin.shouldSendConnectionWarning()) {
```

Reply:
@lego-technix La méthode de répo peut renvoyer "null"

Reply:
OK